examples/graphene_apps/python_worker/avalon_worker/error_code.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import IntEnum, unique
+
+
+@unique
+class WorkerError(IntEnum):
```

Comment:
Instead of this can we use the common/python module, these are already defined there.

Reply:
The idea is to remove any Avalon build dependencies on Python worker and keep it standalone for now. At least till a time when we make a official Avalon SDK release. At that time this python worker is going to install `pip3 install python3-avalon-sdk` :-).

Reply:
Until that is done we can keep using common/python utility. It is good to add TODO for now.